site/netlify/functions/snappi.ts

```diff
@@ -0,0 +1,13 @@
+import createHandler from '@stream-blitz/create-handler';
+
+export const handler = createHandler({
+  name: 'snappi',
+  description: 'That is nice and snappy, it makkes me happy',
+  handler: () => ({
+    audio:
+      'https://res.cloudinary.com/aodhan/video/upload/v1662784292/UntitledAudio_bxn05a.mp4',

```

Comment:
This URL shows up as "This res.cloudinary.com page can’t be found" — maybe permissions need to be updated?

Reply:
I am silly, I forgot about this and deleted those assets, probably get rid of this and I'll probably get around to creating a new one another time.